tests/connectors/anghami.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = function(driver, connectorSpec) {
+	connectorSpec.shouldBehaveLikeMusicSite(driver, {
+		url: 'https://www.anghami.com/explore',
+	});
```

Comment:
The test should start to play the music by itself by specifying 'playButtonSelector' option as CSS selector of an element that will be clicked by WebDriver after URL load to play the music (if the music isn't started automatically). 
Since Anghami doesn't start music automatically, this test won't pass. Here is an example of correct test:
```js
'use strict';

module.exports = function(driver, connectorSpec) {
	connectorSpec.shouldBehaveLikeMusicSite(driver, {
		url: 'https://www.anghami.com/tag/47',
		playButtonSelector: '.track-item .playicon'
	});
};
```
`.track-item .playicon` is just a play button of a first track on the page.